mods/common/chrome/ingame-observer.yaml

```diff
@@ -2,6 +2,26 @@ Container@OBSERVER_WIDGETS:
 	Logic: MenuButtonsChromeLogic
 	Children:
 		LogicKeyListener@OBSERVER_KEY_LISTENER:
+		Container@MUTE_INDICATOR:
```

Comment:
Is this supposed to be here? I don't see any mention of this in the description or comments, and the commit message also doesn't help (i'm sure that the mute icon already works for spectators?)

Reply:
This is a pull request from last year. It seems it is fixed already in current bleed. I removed the commit.